ui/DesignSystem/Component/Wallet/WrongNetwork.svelte

```diff
@@ -1,7 +1,9 @@
 <script lang="typescript">
-  import { Emoji } from "../../Primitive";
+  import * as router from "ui/src/router";
+  import { Button, Emoji } from "ui/DesignSystem/Primitive";
 
   export let expectedNetwork = "Mainnet";
+  export let walletNetwork: string | undefined = undefined;
```

Comment:
I don’t think there’s a reason to make this optional `undefined`. `wallet.connected.network` is always set.

Reply:
26deafc8c49cd7a485fbfbf927590491b44ca99d